pylon-core/src/main/kotlin/io/github/pylonmc/pylon/core/util/position/BlockPosition.kt

```diff
@@ -14,15 +22,18 @@ class BlockPosition(world: World?, val x: Int, val y: Int, val z: Int) {
     val chunk: ChunkPosition
         get() = ChunkPosition(world, x shr 4, z shr 4)
 
-    val asLong: Long
+	@JvmStatic
```

Comment:
```suggestion
	@JvmSynthetic
```
Oops sorry this one's on me

Reply:
Failed to compile, removed the annotation for now